src/lua/CommunityBalanceMod/Tech/TechTreeConstants.lua

```diff
@@ -1,7 +1,18 @@
 local newTechIds = {
     "Resilience",
+    "UpgradeToAdvancedPrototypeLab",
+    "AdvancedPrototypeLab",
+    'ExosuitTech',
```

Comment:
Why are we deleting and re-adding ExosuitTech here?

Reply:
Required as ExosuitTech needs to be > AdvancedPrototypeLab as it's a prerequisite